CHANGELOG.unreleased.md

```diff
@@ -14,6 +14,7 @@
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
 - [SDN Controller] Better detect host shutting down to adapt network topology (PR [#4314](https://github.com/vatesfr/xen-orchestra/pull/4314))
+- [SDN Controller] Fix certificate generator (PR [#4383](https://github.com/vatesfr/xen-orchestra/pull/4383))
```

Comment:
Does this error landed in a XOA release?

Reply:
No, it's on master but not in a XOA release.

Reply:
So I think it might be not relevant here (changelog is based on XOA releases, not changes in `master` because there's no release). A bug triggered and fixed before going into an XOA is not visible for the end user. At least, that's how I imagined it, but maybe @pdonias can give his point of view.

Reply:
Entry removed. :+1:

Reply:
So why is it still there?

Reply:
Rebase error, on it.